src/axom/slam/SubMap.hpp

```diff
@@ -51,10 +51,11 @@ namespace slam
  */
 
 template <typename SuperMapType,
-          typename SubsetType  //= slam::RangeSet<SetPosition, SetElement>
-          >
-class SubMap : public MapBase<typename SubsetType::PositionType>,
-               public SuperMapType::StridePolicyType
+          typename SubsetType,  //= slam::RangeSet<SetPosition, SetElement>
+          typename InterfacePolicy = policies::VirtualInterface>
```

Comment:
Should `InterfacePolicy` be the first template parameter to be consistent with the new code below?

For example:

```
template <typename InterfaceTag, typename PosType, typename ElemType>
struct SetInterfaceSelector
```

Feel free to say no.

Reply:
The template order prioritizes the things that are most likely to change first and the ones that are most likely to use default values last. That allows users to specify fewer template parameters.

I'm not 100% sure about policy vs. tag. Tag makes me think about adding a type as a function parameter rather than a template parameter to the class.

Reply:
Renamed to `InterfacePolicy` for consistency.